### PR TITLE
switch to BeautifulSoup

### PR DESCRIPTION
**Pull request by rnhmjoj**

### CODE REVIEW

**Comment by rnhmjoj** [comment]
Sorry, I have made a mess. It's hopefully fixed now.

File: pirate/torrent.py [hunk 1]
```diff
@@ -76,25 +76,37 @@ def build_request_path(page, category, sort, mode, terms):
...
-    # parse the rows one by one
-    for row in d('table#searchResult tr'):
-        drow = d(row)
-        if len(drow('th')) > 0:
-            continue
+    no_results = re.search(r'No hits\. Try adding an asterisk in '
+                           r'you search phrase\.', html)
+
+    # check for a blocked mirror
+    if not (table or no_results):
```
Comment (vikstrous) [inline pirate/torrent.py]: I think `not table` and `not no_results` is clearer

Comment (rnhmjoj) [inline pirate/torrent.py]: Sure

**Comment by vikstrous** [comment]
awesome! shipit!

**Pull request merged**